ci-operator/step-registry/dpu-operator/workflow/tests/dpu-operator-workflow-tests-commands.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+token_dpu_operator_key=$(cat "/var/run/token/dpu-token/dpu-key")
+endpoint=$(cat "/var/run/token/dpu-token/url")
+
+job_url="https://${endpoint}/job/Lab144_DPU_Operator_Test/lastBuild"
+
+while : 
+do
+    job_check=$(curl -k -s --resolve "${endpoint}:443:10.0.180.88" "$job_url/api/json")
+    result=$(echo "$job_check" | jq -r '.result')
+    if [ $result != "null" ]; then
+	sleep $((RANDOM % 61 + 60))
+        break
+    else
+        sleep 60
+    fi
+done
+
+curl -k --resolve "${endpoint}:443:10.0.180.88" "https://${endpoint}/job/Lab144_DPU_Operator_Test/buildWithParameters?token=$token_dpu_operator_key&branchnumber=$PULL_NUMBER"
```

Comment:
PULL_NUMBER is not defined anywhere?

Reply:
PULL_NUMBER is part of the environment variables in the PROW Pod. We do not need to define it.